R/showcase.R

```diff
@@ -81,7 +81,7 @@ appMetadata <- function(desc) {
 navTabsHelper <- function(files, prefix = "") {
   lapply(files, function(file) {
     with(tags,
-      li(class=if (tolower(file) %in% c("app.r", "server.r", tolower(.globals$appFile))) "active" else "",
+      li(class=if (file %in% .globals$appFile) "active" else "",
```

Comment:
Nit: If `$appFile` is just length-one would it be more clear to just use `identical` or some such instead of `%in%`?